bundles/org.openhab.binding.astro/pom.xml

```diff
@@ -4,14 +4,13 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.openhab.binding</groupId>
-    <artifactId>pom</artifactId>
+    <groupId>org.openhab.addons.bundles</groupId>
```

Comment:
Because the astro `groupId` changed to `org.openhab.addons.bundles` the feature will also need to be updated for this change in:

`features/karaf/openhab-addons/src/main/feature/feature.xml`

Reply:
I am not on PC ATM, but I had been sure that I changed the feature. Will have a look at later

Reply:
Checked it and it should be present already: https://github.com/openhab/openhab2-addons/pull/5018/files#diff-39b45771c05d3a4ec56b70ced2977b0cR49

Reply:
Yes I see it now. :+1: